templates/postfix/main.cf

```diff
@@ -72,6 +77,27 @@ smtpd_recipient_restrictions =
    reject_rbl_client bl.spamcop.net,
    permit
 
+# General TLS security improvements
+# Based on: https://kruyt.org/postfix-and-tls-encryption/
+smtpd_tls_protocols = TLSv1.2, TLSv1.1, !TLSv1, !SSLv2, !SSLv3
```

Comment:
I'd recommend using `smtpd_tls_protocols           = !SSLv2,!SSLv3,!TLSv1` instead to also support `TLSv1.3` (or any future TLS protocols)

Reply:
Agreed, will push an update in a bit